lms/static/js/fixtures/bookmarks/bookmarks.html

```diff
@@ -0,0 +1,10 @@
+<div class="courseware-bookmarks-button">
+    <button type="button" class="bookmarks-button is-inactive" aria-pressed="false">
+        Bookmarks
```

Comment:
Curious why we are adding the font-icon here using CSS rather than the standard ```<i class="" aria-hidden="true"></i>```?  I'm not a fan of injecting any sort of content using CSS.